backend/graminsta/post/views.py

```diff
@@ -1,3 +1,55 @@
-# from django.shortcuts import render
+# -*- coding: UTF-8 -*-
+"""views.py"""
 
-# Create your views here.
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from .services import (create_follow_relationship,
+                       delete_follow_relationship,
+                       get_people_user_follows)
+
+
+class FollowView(APIView):
+    """
+    A class based view to manage follow relationship.
+    """
+    @staticmethod
+    def post(request):
+        """Creates a new follow relationship
+        Parameters
+        ----------
+        request: json format
+            Data containing from_user and to_user
+        Returns
+        -------
+        response: json format
+            Newly created follow relationship or errors
+        """
+        relationship = create_follow_relationship(request.data)
+        return Response(relationship.data, status=status.HTTP_201_CREATED)
+
+    @staticmethod
+    def delete(request):
+        """Deletes an existing follow relationship
+        Parameters
+        ----------
+        request: json format
+            Data containing from_user and to_user
+        """
+        delete_follow_relationship(request.data)
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+    @staticmethod
+    def get(request, username):
```

Comment:
As user has to authenticate themselves in order to call this API, so I think you can get who the current user is by calling `request.user`, instead of passing the username.

Reply:
What I am considering is that maybe the authenticated user wants to have a look at other users' following people, like who are followed by his/her own following people(dispensable feature).

Reply:
I see.